allennlp/data/dataset_readers/nlvr.py

```diff
@@ -26,9 +26,12 @@ class NlvrDatasetReader(DatasetReader):
     instances from text, this class contains a method for creating an agenda of actions that each
     sentence triggers.
 
-    We process the json version of the dataset (http://lic.nlp.cornell.edu/nlvr/) here, to read in
-    the structured representations of the synthetic images instead of dealing with the actual images
-    themselves. The format of each line in the jsonl file is
+    We process the either the original json version of the NLVR dataset
+    (http://lic.nlp.cornell.edu/nlvr/) or the processed version where we group all the worlds that a
+    sentence appears in here.
```

Comment:
You should mention the script that you use to process the data.

Reply:
Done.